src/Data/FileEmbed/Extra.hs

```diff
@@ -6,16 +6,20 @@ import Data.FileEmbed (embedFile)
 import Language.Haskell.TH
 import Path
 import Path.IO
+import System.Environment (lookupEnv)
 import Prelude
 
 embedFileIfExists :: FilePath -> Q Exp
 embedFileIfExists inputPath = do
-  case parseRelFile inputPath of
-    Just path -> do
+  skipEmbedEnvVar <- runIO $ lookupEnv "FOSSA_SKIP_EMBED_FILE_IN_HLS"
+  case (skipEmbedEnvVar, parseRelFile inputPath) of
+    (Just _, _) -> do
+      pure (LitE $ StringL "")
+    (_, Just path) -> do
```

Comment:
This can wait till later, but I think we should refactor a few things here:

- `Q Exp` is an untyped haskell expression, but we should make it a typed haskell expression: `Q TExp`.
- We should split the logical checks of env var and file existence.
- We shouldn't be relying on `OverloadedStrings`, this should work even if that extension is disabled.  (`pure (LitE $ StringL "")` only works because `Bytestring` implements `IsString`)